disconnect-when-backgrounded.js

```diff
@@ -1,18 +1,30 @@
 var disconnectTimer = null;
 
+// 10 seconds by default
+var disconnectTime = (Meteor.settings && Meteor.settings.public && Meteor.settings.public.disconnectTime || 10) * 1000;
```

Comment:
Shouldn't this be read when used inside `disconnectIfHidden()`? This would allow it to be set late after this package has been loaded.